utility/src/main/java/org/oppia/android/util/parser/CustomBulletSpan.kt

```diff
@@ -22,18 +22,22 @@ class CustomBulletSpan(context: Context) : LeadingMarginSpan {
   private var gapWidth: Int = 0
   private var yOffset: Int = 0
   private var bulletLeadingMargin: Int = 0
+  private var textLeadingMargin: Int = 0
+  private var margin: Int = 0
 
   init {
     bulletRadius = context.resources.getDimensionPixelSize(R.dimen.bullet_radius)
     gapWidth = context.resources.getDimensionPixelSize(R.dimen.bullet_gap_width)
     yOffset = context.resources.getDimensionPixelSize(R.dimen.bullet_y_offset)
     bulletLeadingMargin = context.resources.getDimensionPixelSize(R.dimen.bullet_leading_margin)
+    textLeadingMargin = context.resources.getDimensionPixelSize(R.dimen.text_leading_margin)
+    margin = bulletLeadingMargin + textLeadingMargin + bulletRadius * 2
   }
 
   private var mBulletPath: Path? = null
```

Comment:
While here, please move this to be next to the other fields & rename to be just ``bulletPath`` (we don't use Android naming conventions).

Reply:
Done